Roys/.gitignore

```diff
@@ -45,6 +45,7 @@ captures/
 *.iml
 .idea/workspace.xml
 .idea/tasks.xml
+.idea/gradle.xml
```

Comment:
I remember there was a previous PR from someone who removed gradle.xml, and then you added again. Can you guys discuss on this issue and apply the result?